tests/test_permission_to_partial_update_is_supported.py

```
@@ -0,0 +1,188 @@
+        f"/api/gated-authors/{author.id}",
+        content_type="application/json",
+        data={
+            "name": "Patched Author",
+        },
+    )
+
+    assert response.status_code == status.HTTP_404_NOT_FOUND, response.json()
+
+
+@pytest.mark.django_db
+def test_partial_update_permitted_resource_with_permitted_related_resource_should_succeed(
```

Ruff (0.8.2)

Line too long (90 > 88)

(E501)